src/main/java/com/softserveinc/ita/models/ProgressBar.java

```diff
@@ -0,0 +1,21 @@
+package com.softserveinc.ita.models;
+
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Condition.appear;
+import static com.codeborne.selenide.Condition.disappear;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.time.Duration.ofSeconds;
+
+public class ProgressBar {
```

Comment:
no need to create a separate class for it and we already have the same functionality in SpecialitiesPage

better to move these things with the progress bar into MainMenyu class